docs/Development/GoogleCustomSearchConfiguration.md

```diff
@@ -0,0 +1,122 @@
+# Google Custom Search configuration
+
+The February 11, 2020 Open Y release introduces Google Custom Search for the website out of the box.
```

Comment:
Maybe it would be better to specify an Open Y version here but not only date?

Reply:
Makes sense